### PR TITLE
fix: correct scripts linting template name

### PR DESCRIPTION
**Pull request by pevisscher**

### CODE REVIEW

**Comment by pevisscher** [comment]
Current dependencies on/for this PR:
* main
  * **PR #34** <a href="https://app.graphite.dev/github/pr/skyleague/node-standards/34" target="_blank">Graphite</a>  👈
    * **PR #35** <a href="https://app.graphite.dev/github/pr/skyleague/node-standards/35" target="_blank">Graphite</a> 
      * **PR #36** <a href="https://app.graphite.dev/github/pr/skyleague/node-standards/36" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/node-standards/34?utm_source=stack-comment).

**Review by MickVanDuijn — APPROVED** [review]

**Comment by pevisscher** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/node-standards/34?utm_source=stack-comment).


Job Reference: Lq6fAFaN1fLnSbVbBggu

**Pull request merged**
